src/shellbar.scss

```diff
@@ -149,15 +186,38 @@ $block: #{$fd-namespace}-shellbar;
   &__subtitle {
     display: inline-block;
     vertical-align: middle;
+    padding-right: 0;
+    padding-left: $fd-shellbar-item-spacing;
+
+    @include fd-rtl() {
+      padding-right: $fd-shellbar-item-spacing;
+      padding-left: 0;
+    }
+  }
+
+  &__title {
+    font-weight: bold;
+    padding-right: 0;
+    vertical-align: middle;
+
+    @include fd-rtl() {
+      padding-right: $fd-shellbar-item-spacing;
+      padding-left: 0;
+    }
   }
 
   &__logo {
     @include fd-reset();
 
-    display: flex;
-    align-items: center;
-    vertical-align: middle;
-    max-height: fd-space(10);
+    padding-right: 0;
+    padding-left: $fd-shellbar-item-spacing;
+
+    @include fd-rtl() {
+      padding-right: $fd-shellbar-item-spacing;
+      padding-left: 0;
+    }
+
```

Comment:
The logo doesn't have padding-left. All elements are "separated" by `Element Spacing` which is element G in the specs. If there's no Back Button (A1) you don't have padding-left. If there's a Back Button, the Back Button itself should have padding-right playing the role of an Element Spacing
<img width="253" alt="Screen Shot 2020-01-21 at 11 16 45 AM" src="https://user-images.githubusercontent.com/39598672/72822350-f341bf80-3c3f-11ea-9355-85d454ca27e4.png">

Reply:
Should I add an example of a shellbar with a back button?